README.md

```diff
@@ -143,6 +143,12 @@ There are several examples, in the examples directory, that simulate real world
 use-cases. Try running them via the command-line, via `cargo run --release
 --example simple`.
 
+## Logging
+
+If you want logging, enable the `logging` feature flag, and then set RUST\_LOG accordingly.
+See the documentation for [env\_logger](https://docs.rs/env_logger/latest/env_logger/) for specifics.
```

Comment:
You actually don't need the `\` before the `_` when the `_` is surrounded by letters.

Reply:
True, but it doesn't hurt anything either.